src/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg

```diff
@@ -283,6 +283,29 @@ Agents
     SendPilotSubmissionAccounting = True
   }
   ##END
+  ##BEGIN PushJobAgent
+  PushJobAgent
+  {
+    # VO treated (leave empty for auto-discovery)
+    VO =
+    # The DN of the certificate proxy used to submit pilots/jobs. If not found here, what is in Operations/Pilot section of the CS will be used
+    PilotDN =
+    # The group of the certificate proxy used to submit pilots/jobs. If not found here, what is in Operations/Pilot section of the CS will be used
+    PilotGroup =
+
+    # List of sites that will be treated by this PushJobAgent
+    Site = any
```

Comment:
What does it mean "any" here? Any site that is defined in the CS? Please specify even if "obvious".